src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -87,6 +87,34 @@ Blank()
 >> Sum(Table(ParseJSON("[1, 2, 3, 4, 5]")), Value(Value))
 15
 
+>> ForAll(ParseJSON("[5, 6, 7]"), Value(ThisItem))
```

Comment:
>ForAll(ParseJSON("[5, 6, 7]"), Value(ThisItem)) [](http://example.com/codeflow?start=3&length=47)

Another interesting case: `ForAll(ParseJSON("[5, null, 7]"), Value(ThisItem))` - would it return [5,Blank(),7]?

Reply:
It would return [5,Error,7] because we type constructors are "type casts" and cannot convert blank to other types.

Reply:
Ok this was a bug and it has been fixed.